libs/ledger/src/storage_unit/transaction_store_sync_service.cpp

```diff
@@ -463,6 +463,7 @@ TransactionStoreSyncService::State TransactionStoreSyncService::OnResolvingObjec
   if (counts.failed != 0u)
   {
     FETCH_LOG_WARN(LOGGING_NAME, "Lane ", cfg_.lane_id, ": ", "Failed promises: ", counts.failed);
+    pending_objects_.DiscardFailures();
```

Comment:
I believe this should stop it printing out how many failures there are forever.

Reply:
👍